src/main/java/com/myhome/controllers/CommunityController.java

```diff
@@ -205,5 +205,21 @@ public CommunityController(
       return ResponseEntity.status(HttpStatus.NOT_FOUND).build();
     }
   }
+
+  @Operation(description = "Deletion community with given community id")
+  @DeleteMapping(
+          path = "/community/{communityId}",
+          produces = {MediaType.APPLICATION_JSON_VALUE, MediaType.APPLICATION_XML_VALUE},
+          consumes = {MediaType.APPLICATION_JSON_VALUE, MediaType.APPLICATION_XML_VALUE}
+  )
+  public ResponseEntity<GetCommunityDetailsResponse> deleteCommunity(@PathVariable String communityId) {
+    log.trace("Received delete community request");
+    Set<Community> communitiesAfterDeletion = communityService.deleteCommunity(communityId);
+    Set<GetCommunityDetailsResponse.Community> communityDetailsResponse =
+            communityApiMapper.communitySetToRestApiResponseCommunitySet(communitiesAfterDeletion);
+    GetCommunityDetailsResponse response = new GetCommunityDetailsResponse();
+    response.getCommunities().addAll(communityDetailsResponse);
```

Comment:
Let's not return a list of communities in response. Let users `GET` a list by hitting `/communities` if they need to. Can this change be loosely based on the current fix for jmprathab/MyHome#43 (Alert, it is still open 🤫)?

DELETE should ideally return a status code in response without any body.